scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+//@import 'colors';
+
+.tech {
+  width: 200px;
+  height: 300px;
+  padding: 0px 25px;
+  margin: 20px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #BED23C;
```

Comment:
You may extract this color to the constants file

Reply:
This is already done in the branch page-titles.